source/crud/crud.rst

```diff
@@ -407,6 +407,17 @@ Read
 
   class FindOptions {
 
+    /**
+     * Enables writing to temporary files. When set to true, the find operation
```

Comment:
Consider making it clearer that it's the _server_ that's doing this, not the client.

Reply:
Done